src/main/ui/app/route/Show.jsx

```diff
@@ -94,6 +128,10 @@ export default class Show extends Component {
             const requirementmeaning = (
                     <Tooltip id="tooltip"><strong>Minimum version of the OpenMRS Platform required.</strong></Tooltip>
             );
+            let version = this.state.latestModule ?
```

Comment:
I think the best UI here would be for this button to show one of 3 different things:

1. If you're going to get the very latest latest version, it should say: `Download Latest<br/><small>Version 1.2.0</small>`

2. If the latest version supported by your openmrs-core version is not the very latest, say: `Download Latest Supported<br/><small>Version 2.8.0</small>`

3. If no version is supported by your openmrs-core version: have a disabled button that says `No module version supports OpenMRS Core 1.2.0`